README.md

```diff
@@ -1,12 +1,180 @@
+│       ├── ...
+│       ├── src
+│       │   └── components
+│       │       └── catalog
+│       │           ├── ...
+│       │           └── EntityPage.tsx
+│       ├── ...
+│       └──
+└── ...
+```
+
+```typescript
+// imports
+import { IMGitHubDeploymentsDashboard } from '@im-open/im-github-deployments';
+
+...
+
+const deploymentContent = (
+  <Grid container spacing={3} alignItems="stretch">
+    <Grid item xs={12} md={12} lg={12}>
+      <IMGitHubDeploymentsDashboard />
+    </Grid>
+  </Grid>
+};
+
+...
+
+const defaultEntityPage = (
+  <EntityLayout>
+    <EntityLayout.Route path="/" title="Overview">
+      {overviewContent}
+    </EntityLayout.Route>
+
+    <EntityLayout.Route path="/deployments" title="Deployments">
+     {deploymentContent}
+    </EntityLayout.Route>
+
+    <EntityLayout.Route path="/ci-cd" title="CI/CD">
+      {cicdContent}
+    </EntityLayout.Route>
+    ...
+  </EntityLayout>
+);
+
+...
+```
+
+The example adds the `import` statement and adds the `IMGitHubDeploymentDashboard` to the `defaultEntityPage`, but the display of the deployment tab can be based on whatever your catalog considers a deployable entity that is tied to a GitHub workflow.
+
+#### 3. Authentication
+
+The `im-github-dashboard` plugin relies on GitHub authentication.  The user must have a GitHub login and access to the org and repo that is being represented in Backstage Catalog.
+
+![GitHub Authentication](images/github_auth.png)
+
+### Creating deployments via GitHub Workflows and Actions
+
+#### Update deployment workflow to use the [im-open/create-github-deployment] action
+
+After the deployment job, an `update-the-deployment-board` job can report back the results of the deployment.
+
+```yaml
+name: Project Deployment
+on:
+	workflow_dispatch:
+		inputs:
+      tag:
+        description: The tag to deploy
+        type: string
+        required: true
+      environment:
+        description: The environment the branch, tag or SHA was deployed to
+        required: true
+        type: choice
+        options:
+          - Dev
+          - QA
+          - Stage
+          - Prod
+      instance:
+        description: The instance to deploy to
+        required: true
+        type: choice
+        options:
+          - Primary-slot1
+          - Primary-slot2
+          - Secondary-slot1
+          - Secondary-slot2
+
+jobs:
+	deploy-the-project:
+	  outputs:
+	  	deployment_conclusion: ${{ steps.deployment.output.conclusion }}
+		...
+
+	update-the-deployment-board:
+		needs: [deploy-the-project]
+		steps:
+			- name: Create GitHub Deployment
+        id: create-deployment
+        uses: im-open/create-github-deployment@v1
+        with:
+          workflow-actor: ${{ github.actor }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          environment: ${{ input.environment }}
+          release-ref: ${{ input.tag }}
+          deployment-status: ${{ deploy-the-project.outputs.deployment_conclusion }}
+          entity: im-github-dashboard
```

Comment:
```suggestion
          entity: im-github-deployments
```
Is this what you meant? To be consistent with the example used in the prior instructions?